phenol-annotations/src/test/java/org/monarchinitiative/phenol/annotations/formats/hpo/category/HpoCategoryLookupTest.java

```diff
@@ -0,0 +1,47 @@
+package org.monarchinitiative.phenol.annotations.formats.hpo.category;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.monarchinitiative.phenol.io.OntologyLoader;
+import org.monarchinitiative.phenol.ontology.data.Term;
+import org.monarchinitiative.phenol.ontology.data.TermId;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HpoCategoryLookupTest {
```

Comment:
We should have `Arachnodactyly` in `hpo_toy.json`. Can you pls add a test that checks that we assign a correct category to `Arachnodactyly`?